examples/hitl/rearrange_v2/config/language_rearrange_multi_agent.yaml

```diff
@@ -0,0 +1,30 @@
+# This config is derived from habitat-lab/habitat/config/benchmark/multi_agent/hssd_spot_human.yaml
+# @package _global_
+
+defaults:
+  - language_rearrange
+  - /habitat/simulator/agents@habitat.simulator.agents.agent_1: spot
+  - /habitat/simulator/sensor_setups@habitat.simulator.agents.agent_1: rgbd_head_rgbd_arm_agent
+  - /habitat/task/actions@habitat.task.actions.agent_1_arm_action: arm_action
+  - /habitat/task/actions@habitat.task.actions.agent_1_base_velocity: base_velocity
+  - /habitat/task/actions@habitat.task.actions.agent_1_rearrange_stop: rearrange_stop
+  - _self_
+
+habitat:
+  task:
+    actions:
+      agent_1_arm_action:
+        grip_controller: MagicGraspAction
+      agent_1_base_velocity:
+        lin_speed: 40.0
+        ang_speed: 20.0
+  gym:
+    obs_keys:
+      - agent_0_head_depth
+      - agent_0_is_holding
+      - agent_0_ee_pos
+      - agent_1_head_depth
```

Comment:
@zephirefaith Do we need all these sensors?

Reply:
We need the panoptics, we do not need RGB or depth. I can remove those.

Reply:
We also need to make sure that the sensors don't run in single-user and multi-user as these applications run on servers without GPUs.